tests/cancun/eip1153_tstore/test_yul_coverage.py

```diff
@@ -0,0 +1,99 @@
+"""
+Ethereum Transient Storage EIP Tests
+https://eips.ethereum.org/EIPS/eip-1153
+"""
+
+from typing import Dict, Union
+
+import pytest
+
+from ethereum_test_forks import Shanghai
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Environment,
+    StateTestFiller,
+    TestAddress,
+    Transaction,
+)
+from ethereum_test_tools.code import Yul
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "N/A"
+REFERENCE_SPEC_VERSION = "N/A"
+
+
+@pytest.mark.valid_from("Cancun")
+def test_yul_coverage(
+    state_test: StateTestFiller,
+):
+    """
+    Use this test to fill the coverage which is not completed by using Op
+    when converting from .json tests filled with yul and different solc evm versions
+    """
+    address_to = Address("A00000000000000000000000000000000000000A")
+    missed_coverage = Address("B00000000000000000000000000000000000000B")
+    uncalled_account = Address("C00000000000000000000000000000000000000C")
+
+    pre = {
+        address_to: Account(
+            balance=1000000000000000000,
+            nonce=0,
+            code=Yul(
+                """
+            {
+                let ok := call(gas(), 0xB00000000000000000000000000000000000000B, 0, 0, 0, 0, 0)
+                mstore(0, ok)
+                return(0, 32)
+            }
+            """,
+                fork=Shanghai,
+            ),
```

Comment:
```suggestion
            code=Op.MSTORE(0, Op.CALL(Op.GAS, missed_coverage, 0, 0, 0, 0, 0)) + Op.RETURN(0, 32)
```
No real need to use yul here.

Reply:
hm the yul execution triggers one line in the coverage that otherwise would be detected as red